### DISCUSSION TITLE
UseEffect and useLayout hooks are not triggering on safari browsers in "next": "12.1.0"

### DISCUSSION THREAD

**Discussion by Ayesha1212** [vercel/next.js#35924] [Help]
### Verify canary release

- [ ] I verified that the issue exists in Next.js canary release

### Provide environment information

....

### What browser are you using? (if relevant)

Safari

### Describe the Bug

I am calling Rest Api's in useEffect and useLayout hooks, it's working fine in all browsers, creating issue only on safari browser both on desktop and mobile.

### Expected Behavior

This should be work fine in safari, as it's working fine in all other browsers.

### To Reproduce

...

**Comment by icyJoseph** [comment]
Hi,

Do you mean `useEffect` and `useLayoutEffect` are not working in Safari? 

What are you trying to do exactly? Do you think you can share some pseudo-code? Have you tried to reduce the problem to a reproducible case?
